<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainGUI</class>
 <widget class="QMainWindow" name="mainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>SDR Configurations</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <widget class="QPushButton" name="button_load_cfg">
       <property name="geometry">
        <rect>
         <x>228</x>
         <y>30</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>load</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cfg_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>211</width>
         <height>25</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>usrp.cfg</string>
       </property>
       <property name="text">
        <string>usrp.cfg</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>External Configuration file</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_save_cf">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>30</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="button_rx_test_connection">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RX USRP IP:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="indicator_tx_connection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>249</red>
            <green>240</green>
            <blue>107</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>249</red>
            <green>240</green>
            <blue>107</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>249</red>
            <green>240</green>
            <blue>107</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="indicator_rx_connection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>249</red>
            <green>240</green>
            <blue>107</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>249</red>
            <green>240</green>
            <blue>107</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>249</red>
            <green>240</green>
            <blue>107</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TX USRP IP:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tx_ip">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rx_ip">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_tx_test_connection">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>91</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Tx Gain</string>
      </property>
      <widget class="QSlider" name="vslider_tx_gain">
       <property name="geometry">
        <rect>
         <x>27</x>
         <y>30</y>
         <width>41</width>
         <height>81</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-15</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_tx_gain">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>41</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>53</x>
         <y>123</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>dB</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>91</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Rx Gain</string>
      </property>
      <widget class="QSlider" name="vslider_rx_gain">
       <property name="geometry">
        <rect>
         <x>27</x>
         <y>30</y>
         <width>41</width>
         <height>81</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-20</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rx_gain">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>41</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>53</x>
         <y>123</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>dB</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>401</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Tx Carrier Frequency </string>
      </property>
      <widget class="QSlider" name="hslider_tx_fc">
       <property name="geometry">
        <rect>
         <x>59</x>
         <y>30</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1200</number>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
       <property name="value">
        <number>5800</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_tx_fc">
       <property name="geometry">
        <rect>
         <x>159</x>
         <y>50</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>54</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>GHz</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6 GHz</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1.2 GHz</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>401</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Rx Carrier Frequency </string>
      </property>
      <widget class="QSlider" name="hslider_rx_fc">
       <property name="geometry">
        <rect>
         <x>59</x>
         <y>30</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1200</number>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
       <property name="value">
        <number>5800</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rx_fc">
       <property name="geometry">
        <rect>
         <x>159</x>
         <y>50</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>54</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>GHz</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1.2 GHz</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6 GHz</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_tx_pps">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Tx PPS - Source</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="rbutton_tx_pps_ext">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>External</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_tx_pps</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="rbutton_tx_pps_int">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Internal</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_tx_pps</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_tx_ref">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Tx REF - Source</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="rbutton_tx_ref_ext">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>External</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_tx_ref</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="rbutton_tx_ref_int">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Internal</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_tx_ref</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_rx_pps">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Rx PPS - Source</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="rbutton_rx_pps_ext">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>External</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_rx_pps</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="rbutton_rx_pps_int">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Internal</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_rx_pps</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_rx_ref">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Rx REF - Source</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="rbutton_tx_ref_ext_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>External</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_rx_ref</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="rbutton_tx_ref_int_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Internal</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_rx_ref</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QPushButton" name="button_apply_config">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>460</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Signal Configurations</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_sig_config">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>461</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>/home/jr/workspace/uhd/usrp_caa/build/sys_last_params.mat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_load_data">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QPushButton" name="indicator_sig_config">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>97</green>
            <blue>81</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>97</green>
            <blue>81</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>97</green>
            <blue>81</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>System Controls</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>351</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>241</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Transmission</string>
      </property>
      <widget class="QPushButton" name="button_transmit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_tx_stop">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="indicator_tx_in_progress">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>246</red>
             <green>97</green>
             <blue>81</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>246</red>
             <green>97</green>
             <blue>81</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>246</red>
             <green>97</green>
             <blue>81</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_tx_ref"/>
  <buttongroup name="buttonGroup_rx_ref"/>
  <buttongroup name="buttonGroup_rx_pps"/>
  <buttongroup name="buttonGroup_tx_pps"/>
 </buttongroups>
</ui>
